pkg/env/awscnfm_kubeconfig.go

```diff
@@ -13,3 +16,14 @@ const (
 func KubeConfig() string {
 	return os.Getenv(EnvVarKubeConfig)
 }
+
+// CurrentServer returns the server from the current context
+// e.g. https://my.installation.region.provider.company.domain:443
+func CurrentServer(kubeConfigPath string) (string, error) {
```

Comment:
A helper function which I wrote to get the current server from context, e.g. https://my.installation.region.provider.company.domain:443. 

It is used in pkg/action/cl002/ac001/explainer.go to simulate newCrs(server) when using explain.